tests/integration/test_with_portainer_service.py

```diff
@@ -21,9 +65,230 @@ async def test_portainer_connection(
     event_loop: asyncio.AbstractEventLoop,
     portainer_container: tuple[URL, str],
     aiohttp_client_session: ClientSession,
-):
+) -> str:
     portainer_url, portainer_password = portainer_container
 
-    await portainer.authenticate(
+    return await portainer.authenticate(
         portainer_url, aiohttp_client_session, "admin", portainer_password
     )
+
+
+@pytest.fixture
+async def portainer_endpoint_id(
+    loop: asyncio.AbstractEventLoop,
+    portainer_container: tuple[URL, str],
+    aiohttp_client_session: ClientSession,
+    portainer_baerer_code: str,
+) -> int:
+    portainer_url, _ = portainer_container
+
+    endpoint = await portainer.get_first_endpoint_id(
+        portainer_url, aiohttp_client_session, portainer_baerer_code
+    )
+    assert type(endpoint) == int
+    return endpoint
+
+
+async def test_portainer_test_create_stack(
+    loop: asyncio.AbstractEventLoop,
```

Comment:
wrong fixture name. also try without it. if that does not work we need to check why.

Reply:
thx